packages/react-client/src/state.types.ts

```diff
@@ -36,4 +36,4 @@ export type PeerState = {
   tracks: Record<TrackId, Track>;
 };
 
-export type PeerStatus = "connecting" | "connected" | "authenticated" | "joined" | "error" | "closed" | null;
+export type PeerStatus = "connecting" | "connected" | "error" | "idle";
```

Comment:
Should I change `PeerStatus` to `ParticipantStatus` or `ConnectionStatus`?

Reply:
No strong opinion here 🤔 Both seems legit.

Reply:
btw, can we add some comment/other info about difference between connected and idle?